### ISSUE TITLE
Handling Duplicate People

### ISSUE THREAD

**Issue by mlisle** [c21u/blockcerts-admin-app#33]
Here's a scenario:

1. A year ago, Yakut signed up for a blockcert using yakut.gazi@pe.gatech.edu
2. Today, she signed up for a different blockcert using yakut@gatech.edu
3. The admin app kicked off a new email to her, assuming this was a different person.

I don't know the answer to this problem, but it's going to happen alot. Is there a way for us to avoid duplicate records for the same person?

**Comment by stuartf** [comment]
I don't think we can. We can somewhat avoid it by using csv upload and/or #25. Additionally going through the add issuer workflow again shouldn't break anything. We could maybe do something where the user can merge their own accounts by clicking a link in each, but that seems like a pain.
